tests/integration-tests/conftest.py

```diff
@@ -1394,12 +1397,23 @@ def _run_benchmarks(remote_command_executor, scheduler_commands, **kwargs):
                         test_datadir,
                         dimensions,
                     )
+                    # Load reference results to confront benchmark and Assert failure
+                    if check_result:
+                        reference_results = results_csv_to_dict()
+                        check_errors = []
```

Comment:
I'd rename `check_errors`  to `failing_benchmarks`. 

`check` seems an action we want to perform on the errors.